src/link.js

```diff
@@ -19,34 +20,46 @@ const promisify = (func) => () => new Promise((resolve, reject) =>
   func((err, res) => err ? reject(err) : resolve(res))
 );
 
+function promiseWaterfall(tasks) {
+  return tasks.reduce(
+    (prevTaskPromise, task) => prevTaskPromise.then(task),
+    new Promise((r, e) => r())
+  );
+}
+
 const linkDependency = (project, dependency) => {
-  if (project.android && dependency.config.android) {
-    log.info(`Linking ${dependency.name} android dependency`);
-
-    const didLinkAndroid = registerDependencyAndroid(
-      dependency.name,
-      dependency.config.android,
-      project.android
-    );
-
-    if (didLinkAndroid) {
-      log.info(`Android module ${dependency.name} has been successfully linked`);
-    } else {
-      log.info(`Android module ${dependency.name} is already linked`);
+  return new Promise((resolve, reject) => {
```

Comment:
Is it really necessary to wrap it in a promise in this PR? Also - since it's always resolved successfully, I think we can just leave it as is (and add that when we really need async behaviour here)